src/converters/logseq/index.ts

```diff
@@ -0,0 +1,629 @@
+import { exit } from 'process';
```

Comment:
This class uses terms like `roamToIntermediate` which will probably be confusing for people. Would be great if we can get those renamed. 

It also has more bits from the roam converter, like `extractMetaNodeContentAndGetNumRemaningChildren` and dealing with `#`. If these features are present in logseq as well we should probably split out the code or add tests that verify them here.